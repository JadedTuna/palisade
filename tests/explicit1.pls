high x = true;
low y = false;
y = x;