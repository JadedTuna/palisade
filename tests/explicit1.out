[1;35msecurity error: [0minsecure explicit flow
   1 | high x = true;
   2 | low y = false;
   3 | y [1;35m=[0m x;
       [1;35m~~[0m[1;35m^[0m
